### PR TITLE
feat: bcc self for event emails

### LINKED ISSUE #453: Make it possible to add additional emails to bcc when sending event emails

### PR DESCRIPTION
**Pull request by larwaa**
## Proposed Changes

- BCC the user sending event emails

## Types of Changes

- [x] New feature
- [ ] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [ ] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

- Partially resolves #453 

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [ ] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: backend/apps/events/mutations.py [hunk 1]
```diff
@@ -340,13 +340,14 @@ class Arguments:
...
     @login_required
-    def mutate(self, info, event_id, receiverEmails, content, subject):
+    def mutate(self, info, event_id, receiverEmails: list[str], content, subject):
         try:
             event = Event.objects.get(pk=event_id)
         except Event.DoesNotExist:
             raise ValueError("Ugyldig arrangement")
 
         check_user_membership(info.context.user, event.organization)
+        receiverEmails.append(info.context.user.email)
```
Comment (MorganFH) [inline backend/apps/events/mutations.py:350]:
```suggestion
    def mutate(self, info, event_id, receiver_emails: list[str], content, subject):
        try:
            event = Event.objects.get(pk=event_id)
        except Event.DoesNotExist:
            raise ValueError("Ugyldig arrangement")

        check_user_membership(info.context.user, event.organization)
        receiver_emails.append(info.context.user.email)
```

less smelly

File: backend/apps/events/mutations.py [hunk 2]
```diff
@@ -340,13 +340,14 @@ class Arguments:
...
 
         send_event_emails(receiverEmails, content, subject)
```
Comment (MorganFH) [inline backend/apps/events/mutations.py:352]:
```suggestion
        send_event_emails(receiver_emails, content, subject)
```

**Review by MorganFH — APPROVED** [review]
LGTM

**Review by hermanholmoy — APPROVED** [review]
LGTM

**Pull request merged**